src/k4a_ros_device.cpp

```diff
@@ -942,27 +942,31 @@ void K4AROSDevice::framePublisherThread()
   {
     if (k4a_device_)
     {
-      if (!k4a_device_.get_capture(&capture, waitTime))
+      while (!k4a_device_.get_capture(&capture, waitTime))
```

Comment:
one thing we could also do is increase the waitTime. It's currently 4 seconds for the first frame and 1 second afterwards. Maybe the camera is not able to run at 5 fps because of the increasing load :thinking: